ssn/chapters/ModelPropertyInstance.html

```diff
@@ -1,17 +1,40 @@
 <h3>Generic or Specific Instances of sosa:Property</h3>
 <p>
-    The previous edition of SSN [[vocab-ssn-20171019]] left an ambiguity on whether an instance of <a
-        href="#SOSAProperty"><code>sosa:Property</code></a> should be generic to all features of interest (e.g.,
-    <code>ex:Temperature</code>, <code>ex:OnOffStatus</code>), or specific to a single feature of interest (e.g.,
-    <code>&lt;myBodyTemperature&gt;</code>, <code>&lt;LightStatus&gt;</code>).
-    This version solves the ambiguity by differentiating:
+    The SSN Ontology does not provide a general pattern for describing observable or actuatable properties.
+    It is recommended to use entries from one of the many existing catalogues of properties, such as the <a
+        href="https://www.qudt.org/doc/DOC_VOCAB-QUANTITY-KINDS.html">QUDT catalogue of Quantity Kinds</a>, or the <a
+        href="https://vocab.nerc.ac.uk/search_nvs/S06/">BODC parameter semantic model parameter entity names</a>, or the
+    <a href="https://esipfed.github.io/sweet/">SWEET Ontology</a>, or <a
+        href="https://github.com/EnvironmentOntology">The Environment Ontology</a> if at all possible.
+    Alternatively, the <a href="https://w3id.org/env/puv">Parameter Usage Vocabulary</a> or <a
+        href="https://doi.org/10.15497/RDA00071">I-ADOPT</a> may be used to define a new property.
 </p>
-<ul>
-    <li>
-        <a href="#SOSAProperty"><code>sosa:Property</code></a> for the former case, where properties are generic
-    </li>
-    <li>
-        <a href="#SOSAPropertyOfInterest"><code>sosa:PropertyOfInterest</code></a> for the latter case, where
-        properties are tied to a single entity.
-    </li>
-</ul>
+<p>
+    Most commonly an instance of <a href="#SOSAProperty"><code>sosa:Property</code></a> is generic to many features of
+    interest
+    (e.g. <code>ex:Temperature</code>, <code>ex:OnOffStatus</code>).
+    The catalogs mentioned above primarily support this use case.
+    However, a property may be made specific to a single feature of interest using
+    <a href="#SOSAisPropertyOf"><code>sosa:isPropertyOf</code></a>
+    (e.g. <code>ex:SickChildATemperature</code>, <code>ex:LightStatus</code>).
+
+</p>
+<p>
+    For example, the Property <a href="https://qudt.org/vocab/quantitykind/Temperature"><code>qk:Temperature</code></a>
+    from [[QUDT]] may be specialized to a specific feature of interest, such as the temperature of a particular child or
```

Comment:
Specialisation primarily refers to classes. Only with ArchiMate have I seen it used between instances, but even there rarely. Perhaps there is a different way to describe this mechanism?

Reply:
skos:broader/skos:narrower is a standard predicate for generalization/specialization between instances. 
See https://www.w3.org/TR/skos-primer/#sechierarchy 

It is widely recognised that there is crossover between the semantics of skos:Concept and rdfs:Class. 
However, it is also widely recognised that the choice of whether to model a (lower case) concept as a Class or an Individual is generally application specific. Making 'property' a SSN (SOSA) class, and its members as individuals is seen as problematic in some quarters - from the pov of the ISO 19109 General Feature Model this crosses meta-levels uncomfortably. But having 'properties' as individuals does reflect very common usage so that is what we did. But then when we try to model relationships between properties - particularly specialization/generalization - we inevitably hit the issue under discussion here. 

My general preference is to declare modeling property definitions largely 'out of scope' for SSN, since there is a lot of well-used prior work in the space which would be pointless to attempt to replace.

Reply:
OK, but that was not my point, just the wording. I think the two examples are enough to convey these nuances. I just wonder whether using "generalisation" with instances leads to some misunderstanding.